lib/LedgerSMB/Report/Unapproved/Batch_Detail.pm

```diff
@@ -236,6 +236,7 @@ Runs the report, and assigns rows to $self->rows.
 
 sub run_report{
     my ($self) = @_;
+    ###TODO: This should be language for print language options, currently this return country.
```

Comment:
@ehuelsmann
If you can work on this TODO, we can also close 6267 with current PR.
I think, this TODO is an easy one, but I am not able to find how to change. That's why I left.

Reply:
I'm proposing we merge with this TODO open, because it's unrelated to fixing the db->UI binding?

Reply:
Agree, I will create another PR for the TODO with the changes you suggest(when success)